Pengumpulan Tugas 1 - Kelompok 5

faidatul opened the pull request.

faidatul committed:
Pengumpulan Tugas 1 - Kelompok 5

andrisan commented:
Ini commit nya hanya dilakukan satu orang? Kalau bisa semua anggota tim memiliki commit yang berkaitan dengan kontribusinya masing-masing, karena keaktifan akan dilihat dari jumlah commit per orang

faidatul commented:
Maaf pak baru membalas, jadi apakah bisa di resubmit pak? atau ditambahkan
keterangan tambahan saja mengenai pembagian tugasnya? terimakasih pak
sebelumnya.

On Mon, 12 Sept 2022 at 4.25 PM Andri Santoso ***@***.***>
wrote:

> Ini commit nya hanya dilakukan satu orang? Kalau bisa semua anggota tim
> memiliki commit yang berkaitan dengan kontribusinya masing-masing, karena
> keaktifan akan dilihat dari jumlah commit per orang
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/andrisan-ub/pbpu-b-2022/pull/7#issuecomment-1243459134>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUHVKMYDJ4W77VWEEFTEOTLV53ZJJANCNFSM6AAAAAAQKFEXW4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

faidatul committed:
Change Tugas 1 PBPU - UML

faidatul committed:
Penulisan Kasus, Diagram UML serta Kode untuk Pembuatan Class dan Interface

nicolenatsh committed:
Update Tugas 1 PBPU - UML.md

Maria Nicole Natasha Putri Iriandi (205150401111042) - Penjelasan code

nicolenatsh committed:
Revert "Update Tugas 1 PBPU - UML.md"

This reverts commit d136d1a2593221526bc5e84ef71ce3f27646e99d.

faidatul committed:
Tugas 1 Deleted

Recommit Tugas untuk Detail Pembagian Pengerjaan

faidatul committed:
Pembuatan Diagram dan Code PlantUML

Untuk pembuatan Gambar Diagram, pada awalnya dilakukan dengan Draw.io dan dilakukan secara bersama sama oleh seluruh anggota kelompok.

andrisan commented:
Dihapus saja dan diulangi commit+push satu per satu untuk setiap anggota kelompok

faidatul commented:
> Dihapus saja dan diulangi commit+push satu per satu untuk setiap anggota kelompok

baik, terimakasih pak.

nicolenatsh committed:
Update Tugas 1 PBPU - UML.md

Penjelasan kode PlantUML

rahajengyst committed:
Update Tugas 1 PBPU - UML.md

Penjelasan prinsip OOP yang digunakan

erlindaargyy committed:
Update Tugas 1 PBPU - UML.md

Penjelasan panah diagram

ivannaaruth committed:
Update Tugas 1 PBPU - UML.md

Penjelasan Alur
